time_capture/time_capture/doctype/time_capture_settings/time_capture_settings.json

```
@@ -107,13 +108,19 @@
    "fieldtype": "Link",
    "label": "Sick Leave Type",
    "options": "Leave Type"
+  },
+  {
+   "default": "0",
+   "fieldname": "enable_weekly_reminders",
+   "fieldtype": "Check",
+   "label": "Enable weekly Reminders"
```

```suggestion
   "label": "Enable Weekly Reminders",
   "description": "If enabled, Employees with draft Time Captures are reminded weekly to submit these.",
```
- Minor, but widely spread standard: Most (almost all) words in a label start with a capital letter.
- I also added a little description.